website/src/components/Visualisations/Roles/Roles.module.css

```diff
@@ -5,29 +5,25 @@
 
 .root {
     margin: 0;
+    width: 50%;
```

Comment:
this might not look good on mobile

Reply:
but that's really a stretch goal, so nvm. you can however try `min-width: 50%` and see how it looks like

Reply:
Going to leave this for now as I can't see how this graph would work on mobile without rotating (at which point just rotate the device)